fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/NamespaceTools.java

```diff
@@ -75,7 +76,13 @@ public static void validatePath(final Session session, final String path) {
                     try {
                         namespaceRegistry.getURI(prefix);
                     } catch (final NamespaceException e) {
-                        throw new FedoraInvalidNamespaceException("Prefix " + prefix + " has not been registered", e);
+                        if (segment.equals("fcr:backup")) {
```

Comment:
Along the lines of the recommendation that @ajs6f is making, I would suggest updating `FedoraInvalidNamespaceException`'s constructor to take more details (like the offending `segment` and `prefix`) so that in `RepositoryServiceImpl.backupRepository()` you can accurately determine if the issue is with an undefined "fcr:backup" namespace.